### PR TITLE
Update dark-sites.config

### PR DESCRIPTION
**Pull request by SpicyPTV**
More sites that are Dark by default

### CODE REVIEW

**Review by Gusted — APPROVED** [review]
Thanks! LGTM

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -153,6 +153,7 @@ checkra.in
 chess.com
 chessbomb.com
 chitownhousemusic.com
+channel4.com
```
Comment (Gusted) [inline src/config/dark-sites.config:156]:
```
    +   "channel4.com",
        "chat.ryzom.com",
        "cheat.sh",
        "checkra.in",
        "chess.com",
        "chessbomb.com",
        "chitownhousemusic.com",
    -   "channel4.com",
```
This entry needs to be relocated

**Comment by SpicyPTV** [comment]
Resolved the order issue that was found

**Comment by Gusted** [comment]
> Resolved the order issue that was found

It's not resolved yet.

**Comment by SpicyPTV** [comment]
Now it should be fixed. Sorry about that!

**Pull request merged**